packages/ui/src/working-groups/model/getGroup.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiRx } from '@polkadot/api'
+
+import { groupExtrinsics } from '../types'
+
+export function getGroup(api?: ApiRx, groupName?: string) {
```

Comment:
The `groupName` should be proper type IMO:

```suggestion
import { groupExtrinsics, GroupName } from '../types'

export function getGroup(api: ApiRx, groupName: GroupName) {
```